mmdet3d/models/necks/dla_neck.py

```diff
@@ -0,0 +1,214 @@
+import math
+import numpy as np
+from mmcv.cnn import ConvModule, build_conv_layer
+from torch import nn as nn
+
+from mmdet.models.builder import NECKS
+
+
+def fill_up_weights(up):
+    """Simulated bilinear upsampling kernel.
+
+    Args:
+        up (nn.Module): ConvTranspose2d module.
+    """
+    w = up.weight.data
+    f = math.ceil(w.size(2) / 2)
+    c = (2 * f - 1 - f % 2) / (2. * f)
+    for i in range(w.size(2)):
+        for j in range(w.size(3)):
+            w[0, 0, i, j] = \
+                (1 - math.fabs(i / f - c)) * (1 - math.fabs(j / f - c))
+    for c in range(1, w.size(0)):
+        w[c, 0, :, :] = w[0, 0, :, :]
+
+
+class IDAUp(nn.Module):
```

Comment:
IDAUp name is too hard to understand.

Reply:
The IDAup is actually a iterative deep aggregation module, while DLAup is actually a hierarchical deep aggregation module, maybe Iter_Aggregation and Hier_Aggregation respectively?

Reply:
If the module name uses some kind of abbreviation and we want to keep that. At least the summary part should reveal that. For example, we can change the summary in docstring to `Iterative Deep Aggregation (IDA) Upsampling module to upsample features of different scales to a similar scale`

Reply:
IDAUpsample might be better.